yoda.spec

```diff
@@ -11,7 +11,7 @@ Requires: boost python cython
 %prep
 %setup -q -n %{n}/%{realversion}
 
-./configure --prefix=%i --with-boost=${BOOST_ROOT} CXX="$(which %cms_cxx)" CXXFLAGS="%cms_cxxflags"
+./configure --prefix=%i CXX="$(which %cms_cxx)" CXXFLAGS="%cms_cxxflags" --enable-root
```

Comment:
I checked the log and it takes the correct root version from where I have done the build:
/afs/cern.ch/work/x/xjanssen/cms/Rivet/81X_gcc530/a/slc6_amd64_gcc530/lcg/root/6.06.08-dpekbn